### ISSUE TITLE
the `.strict()` method from Xnav is not used

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#493]
A new method has been introduced: https://github.com/volodya-lombrozo/xnav/issues/100#issuecomment-2811862444 Let's use it right after `.path()`, to fail faster.

**Comment by h1alexbel** [comment]
let's wait for https://github.com/volodya-lombrozo/xnav/issues/105

**Comment by h1alexbel** [comment]
@yegor256 I failed to find the places, where `.strict(path, N)` can be introduced. In most cases we expect infinite node set, and in some cases we check how many objects are there, like here:

```java
if (
    xml.path("/object/metas/meta[head='package']").count() == 1L
) {
    name = xml.path("/object/metas/meta[head='package']/tail")
        .findFirst().get().text().get();
} else {
    name = "";
}
```

We do it in two places. However, as you can see, we still can't apply `.strict(path, N)` to it, because of logic in `else` branch. So, I believe we should close this issue. WDYT?
